src/common/src/array/data_chunk.rs

```diff
@@ -746,6 +745,9 @@ impl DataChunkTestExt for DataChunk {
     fn from_pretty(s: &str) -> Self {
         use crate::types::ScalarImpl;
         fn parse_type(s: &str) -> DataType {
+            if let Some(s) = s.strip_suffix("[]") {
+                return DataType::List(Box::new(parse_type(s)));
+            }
```

Comment:
What's this for?

Reply:
to support list type in `DataChunk::from_pretty`.